src/Activity.jsx

```diff
@@ -1,120 +1,356 @@
 import React from 'react';
 import { API } from 'aws-amplify';
 import Chart from 'chart.js';
+import moment from 'moment';
+import Button from '@material-ui/core/Button';
+import Menu from '@material-ui/core/Menu';
+import MenuItem from '@material-ui/core/MenuItem';
+import './Activity.css';
+
+const numOfDays = {
+  1: 31,
+  2: 28,
+  3: 31,
+  4: 30,
+  5: 31,
+  6: 30,
+  7: 31,
+  8: 31,
+  9: 30,
+  10: 31,
+  11: 30,
+  12: 31
+};
 
 class Activity extends React.Component {
   constructor(props) {
     super(props);
     this.state = {
-      rowData: null
+      projectionData: null,
+      karaokeData: null,
+      tongueTwisterData: null,
+      view: 'Days',
+      activity: 'Projection',
+      timeAnchorEl: null,
+      activityAnchorEl: null
     };
   }
 
   async componentDidMount() {
     try {
-      const data = await this.getDecibels();
-      this.setState({ rowData: data });
-      this.draw();
+      const decibels = await this.getDecibel();
+      const proData = decibels.map(val => ({
+        userId: val.userId,
+        createdAt: moment(val.createdAt),
+        avgDecibel: val.avgDecibel,
+        duration: val.duration,
+        decibels: JSON.parse(val.decibels)
+      }));
+      this.setState({ projectionData: proData });
+      const { projectionData, activity } = this.state;
+      this.renderDays(projectionData, activity);
+    } catch (error) {
+      console.log(error);
+    }
+    try {
```

Comment:
```suggestion

    try {
```

And is there any reason you separated to `try` blocks?